stdlib/obj.mli

```diff
@@ -199,3 +199,12 @@ module Ephemeron: sig
   (** Maximum length of an ephemeron, ie the maximum number of keys an
       ephemeron could contain *)
 end
+
+module Effect_handlers : sig
```

Comment:
Why do we need the `Effect_handlers` module in this PR?

Reply:
I thought it was needed, but it turns out it is not. Fixed.